plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -170,6 +172,24 @@ public static long resolveSnapshotId(Table table, long snapshotId)
                 .collect(toImmutableList());
     }
 
+    public static IcebergColumnHandle getPathColumnHandle(int id)
```

Comment:
this belongs rather in IcebergColumnHandle class.

Reply:
thought about that, as done in the datalake connector case, but I felt keeping in `IcebergUtil` will make keep the `IcebergColumnHandle` cleaner and dont have anything `util` kind of function.